backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/database/repos/AcceptingStoresRepository.kt

```diff
@@ -114,17 +117,25 @@ object AcceptingStoresRepository {
             }.firstOrNull()?.let { it[AcceptingStores.id].value }
     }
 
-    fun importAcceptingStores(stores: List<CSVAcceptingStore>, projectId: EntityID<Int>, dryRun: Boolean): StoreImportReturnResultModel {
+    fun importAcceptingStores(stores: List<CSVAcceptingStore>, project: ProjectEntity, dryRun: Boolean): StoreImportReturnResultModel {
         var numStoresCreated = 0
         var numStoresUntouched = 0
+        val projectId = project.id
+        val region = RegionsRepository.findAllInProject(project.project).let {
```

Comment:
Not sure if there is a nicer solution for the social passes.
Another possibility would be to have a `Region_Store_Manager` Role and provide a region when uploading the csv